d2l-table-shared-styles.html

```diff
@@ -53,6 +53,7 @@
 			};
 			--d2l-table-row: {
 				display:table-row;
+				background-color: var(--d2l-table-body-background-color);
```

Comment:
This background color is already applied to the table's body (`<tbody>`), why is it also needed on each row?

Reply:
With sticky columns, the cells bleed through the empty background when they pass over existing content.

![image](https://user-images.githubusercontent.com/15876659/36505877-ec407864-1722-11e8-9b42-44ec251fb43c.png)